_overviews/cheatsheets/index.md

```diff
@@ -594,25 +590,25 @@ println(upper.getOrElse(""))</code></pre></td>
   case Some(x) if !f(x) =&gt; Some(x)
   case _                =&gt; None
 }</code></pre></td>
-      <td>Optional value doesn't satisfy predicate</td>
+      <td>Optional value doesn't satisfy predicate.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>option.exists(f(_))</code></pre>
       <em><strong>same as</strong></em>
       <pre class="highlight"><code>option match {
-  case Some(x) if f(x) =&gt; true
-  case _               =&gt; false
+  case Some(x) =&gt; f(x)
+  case None    =&gt; false
 }</code></pre></td>
-      <td>Apply predicate on optional value or false if empty</td>
+      <td>Apply predicate on optional value or <code>false</code> if empty.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>option.forall(f(_))</code></pre>
       <em><strong>same as</strong></em>
       <pre class="highlight"><code>option match {
-  case Some(x) if f(x) =&gt; true
-  case None            =&gt; false
+  case Some(x) =&gt; f(x)
+  case None    =&gt; true
```

Comment:
Good catch on the bugs with `forall` here!

Again, the code golf version is shorter, but I'm not convinced removing the guard helps.